code/rounds/InProgressRound.cs

```diff
@@ -156,7 +124,7 @@ private TTTTeam IsRoundOver()
         {
             List<TTTTeam> aliveTeams = new();
 
-            foreach (TTTPlayer player in Players)
+            foreach (TTTPlayer player in Utils.GetAlivePlayers())
```

Comment:
TTT also has add-ons like spectator deathmatch, which is a mod that adds the ability to offer dead players the possibility to join a deathmatch while others are still fighting.

With your change, this won't be possible anymore as the specdm player would be alive as well. That's additionally why I'd like to get track of the current player in a round